app/controllers/users_controller.rb

```diff
@@ -5,6 +5,6 @@ def index
 
```

Comment:
- [ ] Kindly fix the `n + 1` problem that exists in the [view](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png) given at requirement which exists in post controller at the index action.
### Hint: you can use the `include` method to fetch all comments on the post.